Upstream-Status: Inappropriate [configuration]

not building doc (doxygen based)

8/19/2010 - created by Qing He <qing.he@intel.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0629b1f..41746cc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,13 +110,6 @@ FIND_PACKAGE(EXPAT REQUIRED)
 FIND_PACKAGE(OpenSSL REQUIRED)
 FIND_PACKAGE(Udev REQUIRED)
 
-FIND_PROGRAM( DOXYGEN doxygen )
-IF ( NOT DOXYGEN )
-  MESSAGE( FATAL_ERROR "doxygen not found: install doxygen to build the documentation." )
-ELSE ( NOT DOXYGEN )
-  MESSAGE( STATUS "doxygen found: ${DOXYGEN}" )
-ENDIF ( NOT DOXYGEN )
-
 MESSAGE(STATUS "soname: ${LIBZYPP_VERSION_INFO}")
 MESSAGE(STATUS "version: ${VERSION}")
 
@@ -154,7 +147,7 @@ ADD_SUBDIRECTORY( devel EXCLUDE_FROM_ALL )
 ADD_SUBDIRECTORY( tools )
 ADD_SUBDIRECTORY( examples )
 ADD_SUBDIRECTORY( po EXCLUDE_FROM_ALL )
-ADD_SUBDIRECTORY( doc )
+#ADD_SUBDIRECTORY( doc )
 ADD_SUBDIRECTORY( vendor )
 ADD_SUBDIRECTORY( tests EXCLUDE_FROM_ALL )
 
